packages/mural-client/src/index.ts

```diff
@@ -169,6 +179,24 @@ export interface ApiClient {
   };
   fetch: FetchFunction;
   url: (path: string) => URL;
+
+  /**
+   * Abort in-flight requests. The request promises will reject with an
+   * AbortError.
+   *
+   * After calling `abort()` you cannot trigger new requests using this instance
+   * of the client. Instead, call `clone()` and trigger requests using the new
+   * instance of the client.
```

Comment:
**docs**: I would add a little bit about the `.clone()` method

```
If you are looking to abort only a subset of requests, first `.clone` the apiClient and run `.abort()` on the corresponding ApiClient instance.
```

Reply:
The limitation is because AbortController and AbortSignal are stateful and cannot be reset once aborted/notified. Subsequent requests that use an aborted signal are aborted immediately.

Some references for this behavior:
1. From the Standard: https://dom.spec.whatwg.org/#abortcontroller-api-integration:
    > Reject immediately if the AbortSignal is already aborted
2. Mozilla AbortController implementation: https://searchfox.org/mozilla-central/rev/e941c8a6d49b4f62f361446de7a80214fb899186/dom/abort/AbortController.cpp
3. Anecdotal evidence: https://stackoverflow.com/questions/51091109/how-can-i-start-another-request-after-abortcontroller-abort

Reply:
@msmolens could the `apiClient` resets its internal AbortController internally instead?